diff/diff.py

```diff
@@ -4,6 +4,7 @@
 
 from deepdiff import DeepDiff
 from diff.moosedeepdiff import MOOSEDeepDiff
+from mooseutils.validate import validate_extension, validate_files_exist
```

Comment:
In mooseutils/__init__.py you can add the following.

```
from .validate import validate_extension, validate_file_exists
```

Then just `import mooseutils` and call `mooseutils.validate_extension` etc. I like to do this so it is clear in the  code where the function is from. Again, I am trying to be consistent with this concept as this moosetools thing moves forward.